packages/react-router/lib/dom/ssr/components.tsx

```diff
@@ -550,7 +550,17 @@ function isValidMetaTag(tagName: unknown): tagName is "meta" | "link" {
  */
 let isHydrated = false;
 
-export type ScriptProps = Omit<
+/**
+  A couple common attributes:
+
+  - `<Scripts crossOrigin>` for hosting your static assets on a different server than your app.
+  - `<Scripts nonce>` to support a [content security policy for scripts](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src) with [nonce-sources](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/Sources#sources) for your `<script>` tags.
+
+  You cannot pass through attributes such as `async`, `defer`, `src`, `type`, `noModule` because they are managed by React Router internally.
+
+  @category Types
+ */
+export type ScriptsProps = Omit<
```

Comment:
this got renamed from `ScriptProps` to `ScriptsProps` to match it's component name (like all the rest)

it wasn't exported before, so not a breaking change